docusaurus/static/recipes-assets/dato-cms/next-with-apollo/next-with-apollo.test.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import AppTree from '../../test-utils/modules/react-app-tree';
+
+beforeEach(() => {
+    process.env.DATOCMS_TOKEN = 'foo';
+});
+
+afterEach(() => {
+    jest.resetModules();
+});
+
+it('should render correctly', () => {
+    const withApollo = require('./next-with-apollo').default;
```

Comment:
Can withApollo be imported at the top?

Reply:
Seems like it runs only once if we import it at the top, right @jdiogopeixoto?  Resolving this.